### ISSUE TITLE
Detekt should detect multiple line ?. operator for nullable items to be merged into one 

### ISSUE THREAD

**Issue by tusharaina** [detekt/detekt#6222]
## Expected Behavior of the rule
```
dialog?.let { 
            it.window?.let { 
                //do something with window
            }
        }
```
Here both dialog and window are nullable. So detekt should give error/warning to merge these two into one statement. Currently detekt does not detect this and we could add new rule to detect this behaviour to merge these two into one statement like below:
```
dialog?.window?.let { 
                //do something with window
            }
        }
```

## Context
This would improve code readability as well as maintainability.

**Comment by BraisGabin** [comment]
Closing in favor of  #6373. I agree that this should be flagged by detekt but it should be handled by `UnnecessaryLet`.
